docs/src/test-components-js.md

```diff
@@ -129,50 +127,53 @@ test('should work', async ({ mount }) => {
 
 <TabItem value="vue">
 
-```js
+```js title="app.spec.ts"
 import { test, expect } from '@playwright/experimental-ct-vue';
 import App from './App.vue';
 
-test.use({ viewport: { width: 500, height: 500 } });
-
 test('should work', async ({ mount }) => {
   const component = await mount(App);
-  await expect(component).toContainText('Vite + Vue');
+  await expect(component).toContainText('Learn Vue');
 });
 ```
 
+```js title="app.spec.tsx"
```

Comment:
I quite liked the "as an alternative" preface that you had. I think we should mention that these two are actually mutually exclusive. We can also do so in the comment inside the JSX version.